.gitignore

```diff
@@ -4,3 +4,4 @@ Gemfile.lock
 *.gem
 coverage/
 tmp/*
+.DS_Store
```

Comment:
Certainly fine to include here, but I also recommend a global gitignore file locally for this :-)